flight/utils/Wrapped_Movement.py

```diff
@@ -0,0 +1,115 @@
+from flight import config
+from flight.utils.latlon import LatLon
+from mavsdk import System
+import mavsdk as sdk
+import logging
+import asyncio
+import math
+
+class D_Move:
+    """
+    Params: Drone, Pylon
+    Calculates and uses gps coordinates of the drone to move to the location of the target pylon 
+    """
+    async def moveTo(self, drone: System, pylon: LatLon):
```

Comment:
I will figure out why this returns true and where it returns to. As of right now I am not sure.

Reply:
It looks like it either returns a `bool` or `None`, but I'm not entirely sure. The function returns true if it gets within range of the target position, but assuming it gets through that for loop (`async for gps in drone.telemetry.position()`) it looks like it would return `None`